martian/src/metadata.rs

```diff
@@ -59,6 +61,22 @@ impl Default for Version {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum ProfileMode {
+    Disable,
+    Cpu,
+    Line,
+    Mem,
+    Perf,
```

Comment:
Not all mode trickle down into the jobinfo.  How it works is when the user puts `--profile=<mode>` on the command line, `<mode>` is looked up in the [`config.json`](https://github.com/martian-lang/martian/blob/84938e9253ba43d522c26f83e90e0c10b600307f/jobmanagers/config.json#L89).  If the value corresponding to that mode has an `adapter` specified, that `adapter` string is passed down to the `jobinfo`.